hack/eksctl-patch.yaml

```diff
@@ -0,0 +1,9 @@
+iam:
+  vpcResourceControllerPolicy: true
+  withOIDC: true
+  serviceAccounts:
+    - metadata:
+        name: efs-csi-controller-sa
+        namespace: kube-system
+      wellKnownPolicies:
+        ebsCSIController: true
```

Comment:
Should here be [efsCSIController](https://eksctl.io/usage/schema/#iam-serviceAccounts-wellKnownPolicies-efsCSIController) instead?

Reply:
Yes, thought I'd done a find and replace everywhere but clearly not!